cron/getIncrementBalance.js

```diff
@@ -0,0 +1,29 @@
+const cron = require('node-cron');
+const { familyModel } = require('../database/models');
+
+const { getLogger } = require('../helpers');
+const logger = getLogger('getIncrementBalance');
+
+async function main() {
+  try {
+    await cron.schedule('1 * * * *', async function () {
+
+      const families = await familyModel.find({});
+
+      families.forEach((item) => {
+        item.balance = item.totalSalary + item.passiveIncome;
+        item.giftsForUnpacked = item.balance / item.flatSquareMeters;
+        return item.save();
```

Comment:
Also this operation is not atomic (your calculations could enter into race condition with other balance or gifts unpacking changes). That's why you should call some of update methods and use `$inc` mongodb operator